features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -1315,8 +1362,11 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                         hint={ isSystemClaim
                             ? t("claims:local.forms.multiValuedSystemClaimHint")
                             : t("claims:local.forms.multiValuedHint") }
-                        readOnly={ isSubOrganization() || isSystemClaim || isReadOnly
-                            || dataType === ClaimDataType.COMPLEX || dataType === ClaimDataType.BOOLEAN
+                        readOnly={
+                            isSubOrganization() || isSystemClaim || isReadOnly ||
+                            dataType === ClaimDataType.COMPLEX || dataType === ClaimDataType.BOOLEAN ||
+                            dataType === ClaimDataType.DATE || dataType === ClaimDataType.DATE_TIME ||
+                            dataType === ClaimDataType.EPOCH
```

Comment:
We may have to reevaluate this. Multi-valued support can be provided for text input. Instead of only validating against the data type, we may be able to check the input type and allow date values when the input format is set to text. But then there will be an inconsistency :(